app/services/instrument_offline_reservation_canceler.rb

```diff
@@ -0,0 +1,44 @@
+class InstrumentOfflineReservationCanceler
+
+  def cancel!
+    reservations_to_cancel.each do |reservation|
+      reservation.transaction do
+        cancel_reservation(reservation)
+        OfflineCancellationMailer.delay.send_notification(reservation)
```

Comment:
Looks like the method is expecting the `id`. Can we use `OfflineCancellationMailer.send_notification(reservation).deliver_later` and start moving towards the ActiveJob syntax? I think if we do that, it'll use global_id automatically.